resources/js/setup-turbo.ts

```diff
@@ -24,6 +24,24 @@ document.addEventListener('turbo:submit-end', (e) => {
   }
 });
 
+document.addEventListener('turbo:before-fetch-response', (e) => {
+  if (!e.detail.fetchResponse.contentType.match(/^text\/osu-turbo-redirect[ ;]*/)) {
```

Comment:
`contentType` is null when replacing an existing user-cover-preset or toggling enabled/disabled (and maybe similar buttons?)

Reply:
well those only need to reload the page, not redirect